packages/gestalt-design-tokens/tokens/vr-theme/comp/spinner.json

```diff
@@ -0,0 +1,34 @@
+{
+  "comp": {
+    "spinner": {
+      "size": {
```

Comment:
This sizes... I'd keep them hard coded cos we don't have sizes right now. We have spacing, but I see spacing more like paddings and margins, more than heights and width.